pages-css/wishlist-css/page-layout.css

```diff
@@ -0,0 +1,136 @@
+@import url("/pages-css/compo-css/navbar.css");
+@import url("/pages-css/compo-css/card.css");
+@import url("/pages-css/wishlist-css/page-style.css");
+@import url("https://fonts.googleapis.com/css2?family=Orienta&family=Quicksand&display=swap");
+
+/* color variables */
+
+:root {
+    --color-footer-background: #cecece;
+    --color-red: #fc0101;
+    --color-white: #ffffff;
+    --color-dark: #050505;
+}
+
+
+/* *** */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Orienta', sans-serif;
+    font-family: 'Quicksand', sans-serif;
+}
+
+
+/* component page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 0.313rem;
+    grid-template-columns: 1fr 3fr 1fr;
+    column-gap: 0.313rem;
+    row-gap: 0.313rem;
+    grid-template-areas: "header header header " "section-1 section-1 section-1 " "section-2 section-2 section-2 " "footer footer footer ";
+    padding: 0.313rem;
+    padding-left: 0.938rem;
+    padding-right: 0.938rem;
+}
```

Comment:
change all these value upto 1 place of decimal. make it in pixels.

Reply:
value changed upto 1 place of decimal